src/Compilers/CSharp/Portable/xlf/CSharpResources.ja.xlf

```diff
@@ -7204,6 +7204,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
         <note />
       </trans-unit>
       <trans-unit id="IDS_CSCHelp">
+        <trans-unit id="IDS_CSCHelp">
```

Comment:
@v-zbsail This extraneous element means the .xlf file is no longer well-formed XML, and our tools are unable to read it.